application/src/main/data/json/edge/instructions/install/centos/instructions.md

```diff
@@ -39,112 +39,92 @@ OpenJDK Runtime Environment (...)
 
-Once PostgreSQL is installed you may want to create a new user or set the password for the main user.
-The instructions below will help to set the password for main PostgreSQL user:
+Install packages and initialize PostgreSQL. The PostgreSQL service will automatically start every time the system boots up.
 
-```text
-sudo su - postgres
-psql
-\password
-\q
+```bash
+sudo dnf -qy module disable postgresql && \
+sudo dnf -y install postgresql16 postgresql16-server postgresql16-contrib && \
+sudo /usr/pgsql-16/bin/postgresql-16-setup initdb && \
+sudo systemctl enable --now postgresql-16
+{:copy-code}
 ```
 
-Then, press "Ctrl+D" to return to main user console.
+Once PostgreSQL is installed, it is recommended to set the password for the PostgreSQL main user.
 
-After configuring the password, edit the pg_hba.conf to use MD5 authentication with the postgres user.
-
-Edit pg_hba.conf file:
+The following command will switch the current user to the PostgreSQL user and set the password directly in PostgreSQL.
 
 ```bash
-sudo nano /var/lib/pgsql/16/data/pg_hba.conf
+sudo -u postgres psql -c "\password"
 {:copy-code}
 ```
 
-Locate the following lines:
-
-```text
-# IPv4 local connections:
-host    all             all             127.0.0.1/32            ident
-```
-
-Replace `ident` with `md5`:
+Then, enter and confirm the password.
 
-```text
-host    all             all             127.0.0.1/32            md5
-```
+Since ThingsBoard Edge uses the PostgreSQL database for local storage, configuring MD5 authentication ensures that only authenticated users or 
+applications can access the database, thus protecting your data. After configuring the password, 
+edit the pg_hba.conf file to use MD5 hashing for authentication instead of the default method (ident) for local IPv4 connections.
 
-Finally, you should restart the PostgreSQL service to initialize the new configuration:
+To replace ident with md5, run the following command:
 
 ```bash
-sudo systemctl restart postgresql-16.service
+sudo sed -i 's/^host\s\+all\s\+all\s\+127\.0\.0\.1\/32\s\+ident/host    all             all             127.0.0.1\/32            md5/' /var/lib/pgsql/16/data/pg_hba.conf
 {:copy-code}
 ```
 
-Connect to the database to create ThingsBoard Edge DB:
+Then run the command that will restart the PostgreSQL service to apply configuration changes, connect to the database as a postgres user, 
+and create the ThingsBoard Edge database (tb_edge). To connect to the PostgreSQL database, enter the PostgreSQL password.
 
 ```bash
-psql -U postgres -d postgres -h 127.0.0.1 -W
+sudo systemctl restart postgresql-16.service && psql -U postgres -d postgres -h 127.0.0.1 -W -c "CREATE DATABASE tb_edge;"
 {:copy-code}
 ```
 
-Execute create database statement:
+#### Step 3. Choose Queue Service
 
-```bash
-CREATE DATABASE tb_edge;
-\q
-{:copy-code}
-```
+ThingsBoard Edge is able to use different messaging systems/brokers for storing the messages and communication between ThingsBoard services. 
```

Comment:
Starting from 4.0 we support only Kafka or in-memory. Please update this msg accordingly.